glacierscript.py

```diff
@@ -88,6 +88,9 @@ def btc_to_satoshi(btc):
     value = btc * 100000000
     return int(value)
 
+def btc_to_mbtc(btc):
+    mbtc = Decimal(btc)*1000
+    return mbtc.quantize(SATOSHI_PLACES)
```

Comment:
Won't this show mBTC with 8 decimal places? Which is a little confusing as mBTC only divides to 5 decimal places.